apps/playground/src/app/quiz/containers/introduction/introduction.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'codelab-quiz-intro',
+  templateUrl: './introduction.component.html',
+  styleUrls: ['./introduction.component.scss']
+})
+export class IntroductionComponent {
+
+  constructor(private router: Router) {}
+
+  startQuiz() {
+    this.router.navigateByUrl('/quiz/question/1');
```

Comment:
Put this in the router config

Reply:
ok, moved to quiz-routing file and added routerLink on start quiz button

Reply:
getting an error trying to load the questions route: ERROR Error: Uncaught (in promise): RangeError: Maximum call stack size exceeded

Reply:
@kirjs, it doesn't seem to want to load the codelab-dependency-injection-quiz component

Reply:
I've changed the routing setup to:
 import('./quiz/quiz-routing.module').then(m => m.QuizRoutingModule), still no luck

Reply:
got it working, now the content-projection/data-binding doesn't seem to be working...

Reply:
trying this, but doesn't work: 
<section id="question">{{ question.questionText }}</section>
<codelab-quiz-question [question]="question"></codelab-quiz-question>

If I add *ngFor="let question of allQuestions" it will work and display all questions and options, shouldn't have to use *ngFor

Reply:
yeah i figured it out, index was initialized to wrong value